scripts/aws/aws_helper.py

```diff
@@ -79,6 +79,14 @@ def get_bucket_name_from_path(path: str) -> str:
 
 
 def parse_path(path: str) -> Tuple[str, str]:
```

Comment:
A tuple doesnt say what is what, so using something that knows that the first part is `bucket` would be easier for people to understand

```python
ret = parse_path("s3://foo/bar")
ret.bucket # foo
ret.key # /bar
```